src/tools/generate/generate_chapters.js

```diff
@@ -11,8 +11,14 @@ converter.setFlavor('github');
 
 const generate_chapters = async () => {
   for (let language of await fs.readdir('content')) {
+    if (ignore(language)) continue;
+
     for (let year of await fs.readdir(`content/${language}`)) {
+      if (ignore(year)) continue;
```

Comment:
Since you're using `readdir` would it ever try to read ds store files as a directory, or is it ignored by default anyway? If anything would we only need to handle these files at leaf?

Even still, I think I agree with @bazzadp that all we need is an `isMarkdown` check on the leaf files since that's the only acceptable input anyway. WDYT?

Reply:
When I was testing using finder to access the content directory, it created `.DS_Store` files in the parent directories too, which was also causing an error.  It seems as if  `readdir` still tries to enumerate the `.DS_Store` as a directory - causing it to break messily.

There does need to be a check at each level, but I am totally open to alternatives.